src/foam/nanos/notification/email/rules.jrl

```diff
@@ -0,0 +1,47 @@
+p({
+  class: "foam.nanos.notification.email.EmailMessagePropertyServiceRule",
```

Comment:
If there is no additional properties or overrides, regular Rule model should work fine
```suggestion
  class: "foam.nanos.ruler.Rule",
```